README.md

```diff
@@ -13,3 +13,25 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+
+Part 1:
+1. Depending on the memory the machine has, or the program acquires, this can slow runtime. For example, if your program uses up a lot of memory, this can slow things down. Like if you had a bunch of arrays, strings, etc your program was looking through and there was not a way to get rid of memory you don't need, this can cause issues down the line.
```

Comment:
But we can analyze memory complexity. I don't understand why this is a reason that asymptotic analysis is misleading.